server/pageRenderer.js

```diff
@@ -120,6 +118,21 @@ export default function pageRenderer({
         ${styles}
       </head>
       <body>
+        <script>
+          ${
+            // If user has selected auto and device is in dark mode; ensure we update
+            // the theme before first render to screen
+            selectedTheme === 'auto'
+              ? `(function () {
+            try {
+              if (window.matchMedia('(prefers-color-scheme: dark)').matches === true) {
```

Comment:
It's also possible to subscribe to changes here, but that is outside the scope here, and others can feel free to do that :ok_hand: